consumers/notifiers/tracing/notifier.go

```diff
@@ -0,0 +1,33 @@
+package tracing
+
+import (
+	notifiers "github.com/mainflux/mainflux/consumers/notifiers"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+const notifierOp = "notifier_op"
+
+var _ notifiers.Notifier = (*serviceMiddleware)(nil)
+
+type serviceMiddleware struct {
+	svc    notifiers.Notifier
+	tracer opentracing.Tracer
+}
+
+// NewNotifier create new notifier tracing middleware service
+func NewNotifier(svc notifiers.Notifier, tracer opentracing.Tracer) notifiers.Notifier {
```

Comment:
```suggestion
func New(svc notifiers.Notifier, tracer opentracing.Tracer) notifiers.Notifier {
```

Reply:
Can't have two functions with the same signature